package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "essote-management-portal",
```

Comment:
Revert this, since property **name** is already defined in package-lock.json row 1.

Reply:
Done.